Harvest.Web/ClientApp/src/Projects/ProjectDetailContainer.tsx

```diff
@@ -78,6 +78,17 @@ export const ProjectDetailContainer = () => {
         <div className="card-content">
           <div className="row justify-content-between">
             <div className="col">
+              <ShowFor
+                roles={["Worker", "Supervisor", "FieldManager"]}
```

Comment:
@srkirkland I don't think workers can see the list of projects to get here.

Reply:
Good point